core/Pos/Binary/Crypto.hs

```diff
@@ -195,11 +194,15 @@ instance (Bi w) => Bi (ProxySignature w a) where
 
 instance Bi PassPhrase where
     put pp = do
+        -- currently passphrase may be 32-byte long, or empty
+        -- (for unencrypted keys)
         let bs = BS.pack $ ByteArray.unpack pp
-        putAssertLength "PassPhrase" passphraseLength bs
+        unless (BS.length bs `elem` [0, 32]) $ error $
+            sformat ("put@PassPhrase: expected length 0 or 32, not "%int)
+                (BS.length bs)
         putByteString bs
     get = label "PassPhrase" $
-          ByteArray.pack . BS.unpack <$> getByteString passphraseLength
+          ByteArray.pack . BS.unpack <$> (getByteString 32 <|> getByteString 0)
```

Comment:
Yep, one day we should )
If you'll make daedalus bridge with `Maybe PassPhrase`, let me know